qa/tasks/radosbench.py

```diff
@@ -76,7 +76,7 @@ def task(ctx, config):
             else:
                 pool = manager.create_pool_with_unique_name(erasure_code_profile_name=profile_name)
 
-        osize = config.get('objectsize', 0)
+        osize = config.get('objectsize', 65536)
```

Comment:
for posterity, rados cli uses `op_size` for `object_size` if the latter is not specified. and `op_size` is `default_op_size` by default.  and `default_op_size` is `1 << 22`.